pyproject.toml

```diff
@@ -65,173 +53,205 @@ py = "importnb.loader:Loader"
 toml = "importnb.loaders:Toml"
 yaml = "importnb.loaders:Yaml"
 yml = "importnb.loaders:Yaml"
-[project.entry-points.pytest11]
-importnb = "importnb.utils.pytest_importnb"
```

Comment:
Oh, this is a pretty big change and probably needs docs, but it was getting hard to reason about what was imported when during the test setup cycle across different pytest versions. 

At least personally I've had the autodiscovery bite me a couple times.

One approach would be to ship a separate `pytest-importnb` package which pretty much just had this line in it and no code. Alternately, there could be some CLI stuff to opt into it, e.g. `pytest --importnb`.

Reply:
#160 mentions this being a problem too. i there might be better ways to import notebook tests like importing notebook in a python test script

Reply:
7c2688c added some approaches to README, and a changelog entry